CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to this project will be documented in this file.
 
 ### :rocket: (Enhancement)
 
+* [#2871](https://github.com/open-telemetry/opentelemetry-js/pull/2871) feat(experimental-packages): Update packages to latest SDK Version. ([@pichlermarc](https://github.com/pichlermarc))
```

Comment:
Can you add sub-bullet points to this? This PR handles a lot more work than what is implied by this title.

Reply:
We can shorten this entry to be like:
```suggestion
* feat(experimental-packages): Update packages to latest SDK Version. #2871 @pichlermarc
```

Reply:
@dyladan Good idea, I added sub-bullet points. in [ecc38bc](https://github.com/open-telemetry/opentelemetry-js/pull/2871/commits/ecc38bc53c5755e83083083820e67497c9cb7c94).

Reply:
I've submitted https://github.com/open-telemetry/opentelemetry-js/pull/2889 to discuss the verbose format.

Reply:
@legendecas yes, this seems way more concise, good idea. :slightly_smiling_face:  I updated the entry to #2889 format in [ab80dba](https://github.com/open-telemetry/opentelemetry-js/pull/2871/commits/ab80dba9a41d1d9030388da0c4e52b3f0bf5bf7a), and moved the entry to the experimental changelog. However, this format does not work for anyone viewing the changelog file locally.